wrappers/msbuild

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright (c) 2024 Sergey Kvachonok
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+. $(dirname $0)/msvcenv.sh
+
+# Prevent MSBuild from messing with PATH, INCLUDE and LIB environment variables.
```

Comment:
Can you extend this sentence a bit, to clarify what "messing with the environment variables" means, so it is understandable even without reading the associated link?

Reply:
Rewritten in https://github.com/mstorsjo/msvc-wine/pull/103/commits/bb1b6767cdfda66a2887648cc749960d247b5b9d#diff-af9afe9da8c6e69fc2391bfcb45a4019e23f6d88f0c42b7f870d923e5c9dd44cR19-R22